features/other_b/skin.feature

```diff
@@ -300,6 +302,7 @@ Feature: creating and editing skins
       And I fill in "CSS" with ".myclass { -moz-box-sizing: border-box; -webkit-transition: opacity 2s; }"
       And I submit
     Then I should see "Skin was successfully created"
+    Then the cache of the skin on "skin with prefixed property" should expire after I save the skin
 
   Scenario: #workskin selector prefixing
     Given basic skins
```

Comment:
This looks good to me, I will be annoying and suggest also checking that a skin cache expires after its parent is saved and that the cache does NOT expire when it shouldn't.